### DISCUSSION TITLE
Word level bounding box not detected in paddle ocr

### DISCUSSION THREAD

**Discussion by Shini1996koshy** [PaddlePaddle/PaddleOCR#8782] [General]
Need word level bounding box for paddleocr . Is there a way to get the word level bounding box identification from paddleocr

**Comment by haooxia** [comment]
Do you know any other algorithms that can implement word-level detection?

**Comment by Shini1996koshy** [comment]
Hi,
I don't have much idea on that . Could you please share if any

Thanks and Regards,
Shini Mary Koshy

**Reply by haooxia** [reply]
I recently found a character segmentation algorithm for ancient Chinese characters called HRCenterNet, and I guess there are some related works for other languages as well.
HRCenterNet: https://github.com/Tverous/HRCenterNet

**Reply by prachii1910** [reply]
do you know any character segmentation algo for english ?

**Reply by haooxia** [reply]
I'm sorry, I haven't searched for that.
